Banyan/Project.toml

```diff
@@ -19,27 +19,22 @@ IniFile = "83e8ac13-25f8-5344-8a64-a9f2b223428f"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 LibGit2 = "76f85450-5226-5b5a-8eaa-529ad045b433"
 MPI = "da04e1cc-30fd-572f-bb4f-1f8673147195"
+MethodAnalysis = "85b6ec6f-f7df-4429-9514-a64bcd9ee824"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
 Serialization = "9e88b42a-f829-5b0c-bbe9-9e923198166b"
+Snappy = "59d4ed8c-697a-5b28-a4c7-fe95c22820f9"
 TOML = "fa267f1f-6049-4f14-aa54-33bafae1ed76"
 TimeZones = "f269a46b-ccf7-5d73-abea-4c690281aa53"
 
 [compat]
-AWS = "1.36.0"
-AWSCore = "0.6.17"
-AWSS3 = "0.7"
-AWSSQS = "0.6.4"
-DataFrames = "1"
-Downloads = "1.4"
-FileIO = "1.9.1"
-FilePathsBase = "0.9.10"
-HTTP = "0.8.19"
-IniFile = "0.5.0"
-JSON = "0.21.1"
-MPI = "0.19.0"
-ProgressMeter = "1.7"
-TimeZones = "1.5.6"
+AWSS3 = "^0.7.7"
+AWSSQS = "^0.6.5"
```

Comment:
Maybe not.. I'll just keep it at 0.7 and 0.6 but I think it's anyway constrained by HTTP.jl (which recently went 1.0) or something like that. In the next release, we should be using AWS.jl for everything (since AWSCore.jl is being deprecated) which should make everything better.